### PR TITLE
env setup for offline build added

### PR DESCRIPTION
**Pull request by sumitnitsurat**
- added lifecycle expiration in serverless.yml.
- Added bucket access to public read on creation
- dotenv to load env variable when running in offline mode

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  setupFiles: ["./envTestSetup"]
```
Comment (rjchow) [inline jest.config.js]: possible to use jest.setup.js?

File: serverless.yml [hunk 2]
```diff
@@ -1,82 +1,100 @@
...
   Resources:
-    DevBucket:
-      Type: AWS::S3::Bucket
-      Properties:
-        BucketName: 'dev-bucket'
     DeployedBucket:
       Type: AWS::S3::Bucket
       Properties:
         BucketName: '${self:custom.BUCKET_NAME.${self:provider.stage}}'
+        AccessControl: PublicRead
```
Comment (rjchow) [inline serverless.yml]: as discussed, only the function needs access to the bucket

File: serverless.yml [hunk 3]
```diff
@@ -1,82 +1,100 @@
...
-            - {http: {path: /storage/queue, method: get, cors: true}}
+      handler: src/storage/queueNumber.handler
+      events:
+        - {http: {path: /storage/queue, method: get, cors: true}}
+        - s3:
+          bucket: ${self:provider.environment.BUCKET_NAME}
+          event: s3:ObjectCreated:*
+          lifecycle:
+            -
+              ExpirationInDays: 2
```
Comment (rjchow) [inline serverless.yml]: we're hardcoding the ttl?

File: serverless.yml [hunk 4]
```diff
@@ -1,82 +1,100 @@
...
     getQueueNumber:
-        handler: src/storage/queueNumber.handler
-        events:
-            - {http: {path: /storage/queue, method: get, cors: true}}
+      handler: src/storage/queueNumber.handler
+      events:
+        - {http: {path: /storage/queue, method: get, cors: true}}
+        - s3:
+          bucket: ${self:provider.environment.BUCKET_NAME}
+          event: s3:ObjectCreated:*
```
Comment (rjchow) [inline serverless.yml]: where will we use this event?

Comment (sumitnitsurat) [reply serverless.yml]: removed

File: serverless.yml (hunk 2 above)
Comment (sumitnitsurat) [reply serverless.yml]: removed

File: jest.config.js (hunk 1 above)
Comment (sumitnitsurat) [reply jest.config.js]: dont need this.

File: serverless.yml (hunk 3 above)
Comment (sumitnitsurat) [reply serverless.yml]: added dynamic OBJECT_TTL env.

**Pull request merged**